lib/utils/stackTraceParser.js

```diff
@@ -83,9 +83,11 @@ function stripSuitestCodeFromStack(stack) {
 }
 
 function stripAbsolutePaths(stack) {
-	const cwdInStack = escapeRegexp('(' + process.cwd()) + '(/|\\\\)';
+	const cwdInStack = escapeRegexp(process.cwd()) + '(/|\\\\)';
```

Comment:
I just noticed that we use process.cwd(), and I if user run suitest command somewhere outside project folder, stack trace will be generates correct?

Reply:
in `suitest.interactive()` we change working dir to either a provided value or dirname of the file containing the call. Or do you have something else in mind?

Reply:
I talk not only about repl mode. What if I will run full specified suitest cli command from system root (C:\ for example) for test which located somewhere in C:\User\Development\testCases\test.js, what stack trace will be generated if I will catch error in this case? Unfortunately I never check this case.